docs/docs-content/clusters/cluster-management/ssh-keys/ssh-usernames.md

```diff
@@ -0,0 +1,74 @@
+---
+sidebar_label: "SSH Usernames"
+title: "SSH Usernames"
+description:
+  "A list of the Secure Shell (SSH) usernames created on Kubernetes nodes for each provider and operating system pack
+  available in Palette."
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 11
+---
+
+This page lists the Secure Shell (SSH) user names created on Kubernetes nodes, which vary by provider and operating
+system.
+
+These user names are relevant when using [SSH key pairs](./ssh-keys.md) with your Palette host clusters.
+
+## Public Cloud
+
+### Infrastructure Provider
+
+| Provider       | Operating System Pack | SSH Username |
+| -------------- | --------------------- | ------------ |
+| **AWS IaaS**   | **Ubuntu**            | `ubuntu`     |
+| **AWS IaaS**   | **CentOS**            | `centos`     |
+| **Azure IaaS** | **Ubuntu**            | `ubuntu`     |
+| **Azure IaaS** | **CentOS**            | `centos`     |
+| **GCP IaaS**   | **Ubuntu**            | `ubuntu`     |
+| **GCP IaaS**   | **CentOS**            | `centos`     |
+
+### Managed Kubernetes
+
+| Provider      | Operating System Pack          | SSH Username |
+| ------------- | ------------------------------ | ------------ |
+| **AWS EKS**   | **Amazon EKS optimized Linux** | `ec2-user`   |
+| **Azure AKS** | **Linux**                      | `azureuser`  |
+| **GCP GKE**   | **Container-Optimized OS**     | N/A          |
```

Comment:
N/A seems a bit contradictory to the section below. Maybe we can say something like? "Refer to the SSH Configuration on GCP GKE section"